### DISCUSSION TITLE
v1.6.4

### DISCUSSION THREAD

**Discussion by askdkc** [askdkc/breezejp#91] [Announcements]
### Laravelのページネーションの日本語化に対応しました👍

* fix readme by @askdkc in https://github.com/askdkc/breezejp/pull/89
* Add Pagination Translation by @askdkc in https://github.com/askdkc/breezejp/pull/90

**Full Changelog**: https://github.com/askdkc/breezejp/compare/v1.6.3...v1.6.4

<hr /><em>This discussion was created from the release <a href='https://github.com/askdkc/breezejp/releases/tag/v1.6.4'>v1.6.4</a>.</em>

**Comment by TYMtwilight** [comment]
くらやいな

**Reply by askdkc** [reply]
> くらやいな

おっと？意味がよく分からない👀
